### PR TITLE
Milestone 5 - Calculation

### PR DESCRIPTION
**Pull request by omar-labana**
Highlights on this pull request:
- Edit calculate and operate modules
- Add logic to App component using calculate and operate modules
- Pass and accept event handlers from App component to each button

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,16 +1,44 @@
...
 
+// function App() {
+//   return (
+//     <>
+//       <Display />
+//       <ButtonPanel />
+//     </>
+//   );
+// }
+
```
Comment (george-shammar) [inline src/components/App.js:43]: - [ ] The App component should have` handleClick` implemented. You've commented out the binding to `this` in the constructor.
- [ ] You have snippets of codes commented out in your codebase. Please, let's clean it up.
- [ ] Your application has some unusual behaviours:
- `Null` is being displaced after clicking `AC` button. This should be `0`. Please let's fix it.
- `X` operator displays syntax error. Multiplication does not function.
- `÷` operator displays syntax error. Division does not function.
- There's no number `9` on your calculator.
- `%` operator displays an error page with `Error: [big.js] Invalid number`.
- Decimal `.` does not work.
I'll advise that you check out your `operate.js` and `calculate.js`. Please let's address all these issues.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @omar-labana , 

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by omar-labana** [comment]
@george-shammar thanks mate, I have fixed all the cases you mentioned.

**Review by oloomoses — APPROVED** [review]
Hi @omar-labana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**